### PR TITLE
Reveal managed account APIs

### PR DESCRIPTION
**Pull request by jpjoyal**
@mspensieri @alavers @spasiu @jugarrit

### CODE REVIEW

File: source/includes/rest/_managed.md [hunk 1]
```diff
@@ -359,7 +330,7 @@ Returns an app-scoped JWT signed using the requested keyId/secret pair.
 
 This set of endpoints is used to configure and manage various front-end messaging channels. A JWT is required with `appMaker` scope. The paths below assume that the call will be made using an 'appMaker' scoped JWT.
 
-The currently supported integration types are: Facebook Messenger, LINE, Telegram, Twilio SMS, WeChat, and Email.
+The currently supported integration types are: Facebook Messenger, LINE, Telegram, Twilio SMS, WeChat, Viber and Email.
```
Comment (dannytranlx) [inline source/includes/rest/_managed.md]: That's kinda a pain to maintain, maybe we could link to a marketplace category that could group all of them?

File: source/includes/rest/_managed.md [hunk 2]
```diff
@@ -1,46 +1,17 @@
...
-### Using the App Maker Scoped Token
-
-Use the appMaker scoped Token in the same way that you would use an app or appUser scoped JSON Web Token, by passing it as an Authorization header, with the token prefaced by the _Bearer_ keyword.
+Managed Accounts require a `account` scoped JWT. To sign those JWTs, you will need  use an Account level secret key. You create those by going into your [account page](https://app.smooch.io/account).
 
-All of the existing app management APIs such as /v1/appusers and /v1/webhooks will be accessible using an appMaker JWT, provided the app id is included in the front of the path.
+### Using the Account Scoped Token
 
-> Create a webhook:
+Use the account scoped JWT in the same way that you would use an app or appUser scoped JWT, by passing it as an Authorization header, with the token prefaced by the _Bearer_ keyword.
```
Comment (dannytranlx) [inline source/includes/rest/_managed.md]: I would add a link to the `Authentification` section for more details

File: source/includes/rest/_managed.md [hunk 3]
```diff
@@ -1,46 +1,17 @@
...
-> Fetch a conversation:
-
-```shell
-curl https://api.smooch.io/v1/apps/55c8d9758590aa1900b9b9f6/appusers/c7f6e6d6c3a637261bd9656f/conversation \
-    -H 'authorization: Bearer your-appmaker-token'
-```
-
-```javascript
-// These endpoints are not currently wrapped in a JavaScript lib
-```
```
Comment (dannytranlx) [inline source/includes/rest/_managed.md]: Why don't we keep the code snippets? It helps with the confusing sentence above!

File: source/includes/rest/_managed.md [hunk 4]
```diff
@@ -359,7 +330,7 @@ Returns an app-scoped JWT signed using the requested keyId/secret pair.
 
 This set of endpoints is used to configure and manage various front-end messaging channels. A JWT is required with `appMaker` scope. The paths below assume that the call will be made using an 'appMaker' scoped JWT.
```
Comment (dannytranlx) [inline source/includes/rest/_managed.md]: This should be updated to `account` JWT.

Comment (dannytranlx) [inline source/includes/rest/_managed.md]: More appMaker JWT instance show up in code snippets further down the file.

File: source/includes/rest/_managed.md [hunk 5]
```diff
@@ -1,46 +1,17 @@
...
-Software vendors will be issued a special JWT access token with appMaker scope. The Inside API endpoints defined in this section can only be accessed with appMaker scope.
-
-### Acquiring your App Maker Scoped Token
-
-To acquire the app maker scoped token, visit our [partner page](https://smooch.io/partners/).
-
-### Using the App Maker Scoped Token
-
-Use the appMaker scoped Token in the same way that you would use an app or appUser scoped JSON Web Token, by passing it as an Authorization header, with the token prefaced by the _Bearer_ keyword.
+Managed Accounts require a `account` scoped JWT. To sign those JWTs, you will need  use an Account level secret key. You create those by going into your [account page](https://app.smooch.io/account).
```
Comment (dannytranlx) [inline source/includes/rest/_managed.md]: 2 spaces after `you will need`

Comment (alavers) [inline source/includes/rest/_managed.md]: require an `account` scoped JWT

Comment (alavers) [inline source/includes/rest/_managed.md]: Also, you accidentally a word

File: source/includes/rest/_managed.md (hunk 3 above)
Comment (jpjoyal) [inline source/includes/rest/_managed.md]: They actually looked out of place and I think the e.g help better in making that clear

File: source/includes/rest/_managed.md (hunk 5 above)
Comment (jpjoyal) [inline source/includes/rest/_managed.md]: changed all appMaker to account

Comment (jpjoyal) [inline source/includes/rest/_managed.md]: done

File: source/includes/rest/_managed.md [hunk 6]
```diff
@@ -1,46 +1,17 @@
...
-Software vendors will be issued a special JWT access token with appMaker scope. The Inside API endpoints defined in this section can only be accessed with appMaker scope.
-
-### Acquiring your App Maker Scoped Token
-
-To acquire the app maker scoped token, visit our [partner page](https://smooch.io/partners/).
-
-### Using the App Maker Scoped Token
-
-Use the appMaker scoped Token in the same way that you would use an app or appUser scoped JSON Web Token, by passing it as an Authorization header, with the token prefaced by the _Bearer_ keyword.
+Managed Accounts require a `account` scoped JWT. To sign those JWTs, you will need to use an Account level secret key. You can create one by going into your [account page](https://app.smooch.io/account).
```
Comment (mspensieri) [inline source/includes/rest/_managed.md]: `https://jpjoyal.ngrok.io/rest#jwt` lol

Comment (jpjoyal) [inline source/includes/rest/_managed.md]: :face_palm:

Comment (jpjoyal) [inline source/includes/rest/_managed.md]: fixed

**Pull request merged**